packages/fx-core/src/component/constant/commonConstant.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export class HttpStatusCode {
+  public static readonly HTTP_OK_RESPONSE_CODE = 200;
```

Comment:
I think it is enough to just use keyword for the http status code, e.g. "HTTP_OK_RESPONSE_CODE" -> "OK", so that in code we can just use HttpStatusCode.OK for that value. You can find the variable name from this https://developer.mozilla.org/en-US/docs/Web/HTTP/Status, such as "OK", "Created", "Accepted".

Another question is what about using enum instead of constant?